modules/intentIqAnalyticsAdapter.js

```diff
@@ -149,15 +146,19 @@ function bidWon(args) {
   if (iiqAnalyticsAnalyticsAdapter.initOptions.lsValueInitialized && !iiqAnalyticsAnalyticsAdapter.initOptions.lsIdsInitialized) { initReadLsIds(); }
   if (!iiqAnalyticsAnalyticsAdapter.initOptions.manualReport) {
     ajax(constructFullUrl(preparePayload(args, true)), undefined, null, { method: 'GET' });
+    logInfo('IIQ ANALYTICS -> BID WON')
+    return true
   }
 
-  logInfo('IIQ ANALYTICS -> BID WON')
 }
 
 function getRandom(start, end) {
   return Math.floor((Math.random() * (end - start + 1)) + start);
 }
 
+const intentIqBidWon = { reportExternalWin: bidWon }
+window.pbjs.intentIqBidWon = intentIqBidWon
```

Comment:
Pls make this function accessible via `window.pbjs.intentIqAnalyticsAdapter.reportExternalWin(someData)`. Additionally, if there's a way to use `globalVarName`, please implement it, as the name may vary from `pbjs` in certain cases.